samples/net/openthread/coap_client/README.rst

```diff
@@ -63,6 +99,8 @@ This sample uses the following |NCS| libraries:
 
 In addition, it uses the following Zephyr libraries:
 
+* :ref:`CoAP <zephyr:coap_sock_interface>`
+
 * :ref:`zephyr:kernel`:
 
   * ``include/kernel.h``
```

Comment:
I would add here a references section with reference to [CoAP specification](https://tools.ietf.org/html/rfc7252)

Reply:
I would not add this here, but rather in the sample's overview.
This section lists only dependencies from NCS and Zephyr, to my best knowledge.